source/Calamari.Tests/KubernetesFixtures/Authentication/SetupKubectlAuthenticationFixture.cs

```diff
@@ -181,6 +181,36 @@ public void Execute_WithUsernamePasswordAndCertificates_ConfiguresAuthentication
             }, opts => opts.WithStrictOrdering());
         }
 
+        [Test]
+        public void Execute_WithOnlyCertificates_ConfiguresAuthenticationCorrectly()
```

Comment:
I missed this case in the previous PR. This case is when you provide now account type but you do provide a set of certificates.